dask/dataframe/methods.py

```diff
@@ -134,14 +134,87 @@ def var_aggregate(x2, x, n, ddof):
 
 
 def describe_aggregate(values):
-    assert len(values) == 6
-    count, mean, std, min, q, max = values
+    assert len(values) > 0
+
+    # arrange categorical and numeric stats
+    names = []
+    values_indexes = sorted((x.index for x in values), key=len)
+    for idxnames in values_indexes:
+        for name in idxnames:
+            if name not in names:
+                names.append(name)
+
+    return pd.concat(values, join_axes=pd.Index([names]), axis=1)
```

Comment:
`names` is already a list, which makes `pd.Index([names])` an index of a nested list. This is an odd pattern for me, is this necessary? Can you not just pass `names` to `join_axes`?

Reply:
@jcrist to be honest this line is a copy-paste from pandas source code. I looked into docs - https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.concat.html and it says that join_axes is a list of index objects. I changed it to `join_axes=[pd.Index(names)]` so that it does not look confusing.